packages/lodestar-cli/package.json

```diff
@@ -65,6 +65,7 @@
     "deepmerge": "^3.2.0",
     "ethers": "^5.0.2",
     "expand-tilde": "^2.0.2",
+    "got": "^11.5.0",
```

Comment:
got is a lightweight and modern npm client already used in previous lodestar-cli dependencies